### PR TITLE
Add XCM Tools follow-up

### PR DESCRIPTION
**Pull request by gmajor-encrypt**
#### Project Abstract

> Please replace these instructions with a brief description of your project summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [x] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ Polkadot (USDC & USDT) address in the application).
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

File: applications/xcm-tools-follow-up.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+### Milestone 1
+
+* **Estimated duration:** 4 week
+* **FTE:**  1
+* **Costs:** 9,000 USDC
+
+| Number | Deliverable               | Specification                                                                                                                                                   |
+|-------:|---------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                   | MIT or Apache 2.0                                                                                                                                               |
```
Comment (semuelle) [inline applications/xcm-tools-follow-up.md:99]: Isn't xcm-tools Apache 2.0?

File: applications/xcm-tools-follow-up.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+# XCM Tools follow-up
+
+* **Team Name:** [gmajor](https://github.com/gmajor-encrypt)
+* **Payment Address:** 0xC3094f0ddce699a1Ad9Ef2621DF68Cd297a4c44F(USDC)
```
Comment (semuelle) [inline applications/xcm-tools-follow-up.md:4]: Please change this to an AssetHub address. We no longer support USDC on Ethereum.

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the application. Could you submit your KYC data through [this link](https://in.sumsub.com/idensic/l/#/uni_mjcB05gJZfh8dbr7) (assuming you are applying as an individual; otherwise, please use [this link](https://in.sumsub.com/idensic/l/#/uni_JRqwo72Y5AjXaRDO))?

**Comment by gmajor-encrypt** [comment]
> Thanks for the application. Could you submit your KYC data through [this link](https://in.sumsub.com/idensic/l/#/uni_mjcB05gJZfh8dbr7) (assuming you are applying as an individual; otherwise, please use [this link](https://in.sumsub.com/idensic/l/#/uni_JRqwo72Y5AjXaRDO))?

Thanks. I have submited my KYC data.

**Review by keeganquigley — CHANGES_REQUESTED** [review]
Thanks for the application @gmajor-encrypt you've provided good deliveries for us in the past, but I'm not seeing that this project has been [well-maintained](https://github.com/gmajor-encrypt/xcm-tools/graphs/code-frequency) over the past year, nor if it has gained much attention from the community yet, and there is only one release. In building the tool out, what are some ways you would look to increase dev engagement with the tool?

**Comment by gmajor-encrypt** [comment]
Hi, @keeganquigley 
The main reasons for the lack of maintenance in the past 5 months are as follows:
1. The xcm protocol has only been updated once in the past few months. I also discovered compatibility issues after the upgrade during recent use, and will support it after this update.
2. It is true that this project has not received much attention from developers, and has not received any issues or feature requests. In addition, I have not encountered any obvious problems during use.

> In building the tool out, what are some ways you would look to increase dev engagement with the tool?
I think improvements can be made in these:
1. I used github to search for 'polkadot xcm' and did not find this project. This will cause developers looking for related open source tools to be unable to find ``xcm-tools``. I will modify the project's description & website & topics to improve Matches for github/google searches.

2. Provide code examples (links to projects) on some questions about xcm in forum/Stack Overflow to improve interactivity with developers.

File: applications/xcm-tools-follow-up.md (hunk 1 above)
Comment (semuelle) [reply applications/xcm-tools-follow-up.md]: Just want to make sure: are you planning to publish the tools under MIT license? Looks like you just removed mention of Apache.

Comment (gmajor-encrypt) [reply applications/xcm-tools-follow-up.md]: @semuelle Sorry, I always thought xcm-tools was under the MIT license. It has been modified now.

**Review by keeganquigley — APPROVED** [review]
Thanks for the explanations @gmajor-encrypt given the price and your previous work I'm willing to support it. After this I think it would be good to try to gain some traction in the community and go for treasury funding.

**Review by semuelle — APPROVED** [review]
Thanks for the updates, @gmajor-encrypt.

**Pull request merged**